mat64/lu.go

```diff
@@ -68,14 +69,29 @@ func (lu *LU) Factorize(a Matrix) {
 	lu.updateCond(anorm)
 }
 
-// Det returns the determinant of the matrix that has been factorized.
+// Det returns the determinant of the matrix that has been factorized. In many
+// expressions, using LogDet will be more numerically stable.
 func (lu *LU) Det() float64 {
+	det, sign := lu.LogDet()
+	return math.Exp(det) * sign
+}
+
+// LogDet returns the log of the determinant and the sign of the determinant
+// for the matrix that has been factorized. Many expressions multiply or divide
```

Comment:
I'm not sure that this needs to be spelled out, but:

"Numerical stability in product and division expressions is generally improved by working in log space."